tcms/testruns/migrations/0007_auto_20191004_2053.py

```diff
@@ -0,0 +1,109 @@
+# Generated by Django 2.2.6 on 2019-10-04 20:53
+
+from django.db import migrations, models
+
+from tcms.testruns.models import TestExecutionStatus
+
+
+def insert_default_values(apps, schema_editor):
+    failed = TestExecutionStatus(
+        name='FAILED',
+        status=TestExecutionStatus.NEGATIVE,
+        is_completed=True,
+        chart_name='failed',
+        icon='fa fa-times-circle-o',
+        color='#cc0000',
+        color_code='fail'
+    )
+    failed.save()
```

Comment:
This is duplicating values in the database. Previous migration already adds records with several predefined names. This migration needs to find and update them, not add them a second time.